project-helper.ts

```diff
@@ -55,5 +56,15 @@ export class ProjectHelper implements IProjectHelper {
 		let sanitizedName = _.filter(appName.split(""), (c) => /[a-zA-Z0-9]/.test(c)).join("");
 		return sanitizedName;
 	}
+	
+	private isProjectFileCorrect(projectFilePath: string): boolean {
+		if(this.$staticConfig.CLIENT_NAME_KEY_IN_PROJECT_FILE) {
+			var fileContent = this.$fs.readJson(projectFilePath).wait();
```

Comment:
Actually when we use this function we are sure that the file exists - https://github.com/telerik/mobile-cli-lib/pull/329/files#diff-2fc55d29fa5bf8690e0a4ae30a71effdR25